test/cypress/e2e/search-form.cy.ts

```diff
@@ -0,0 +1,214 @@
+// eslint-disable-next-line import/extensions
+import reactSelectSelectors from '../fixtures/react-select-selectors.json';
+
+const accountHash =
```

Comment:
Could we put these into another file please?

Reply:
Relocated!

Reply:
Relocated!